djangocms_pageadmin/static/djangocms_pageadmin/js/actions.js

```diff
@@ -4,6 +4,120 @@
     }
 
     $(function() {
+
+        var createBurgerMenu = function(row) {
+
+            // create anchor icon
+            var anchor = document.createElement('A');
+            var cssclass = document.createAttribute('class');
+            cssclass.value = 'btn cms-page-admin-action-btn closed';
+            anchor.setAttributeNode(cssclass);
+            var title = document.createAttribute('title');
+            title.value = 'Actions';
+            anchor.setAttributeNode(title);
+            var icon = document.createElement('IMG');
+            var src = document.createAttribute('src');
+            src.value = url + 'svg/menu.svg';
```

Comment:
Couldn't see `url` defined anywhere

Reply:
Besides, do we have to create these elements in js? Can't this be done in a template?

Reply:
This way you won't have to pass that prefix to js

Reply:
Too much work. The idea here was to keep it quick and simple with a javascript solution.